cpp/src/groupby/sort/group_reductions.hpp

```diff
@@ -442,6 +442,76 @@ std::unique_ptr<column> group_merge_m2(column_view const& values,
                                        rmm::cuda_stream_view stream,
                                        rmm::mr::device_memory_resource* mr);
 
+/**
+ * @brief Generate a tdigest column from a grouped set of numeric input values.
+ *
+ * The tdigest column produced is of the following structure:
+ *
+ * list {
+ *   struct {
+ *     double    // mean
+ *     double    // weight
+ *   }
+ * }
+ *
+ * Each output row is a single tdigest.  The length of the row is the "size" of the
+ * tdigest, each element of which represents a weighted centroid (mean, weight).
```

Comment:
This is helpful description. Something like this should go on the aggregation docs.